src/test-support/mockapi.js

```diff
@@ -40,7 +40,7 @@ function makeMockApiRouter(opts) {
     const verb = (req.body.verb || 'get').toLowerCase();
     const path = stripTrailingSlash(req.body.path);
 
-    opts.logger.info(`mock: ${auth} ${verb} ${path}`);
+    opts.logger.verbose(`mock: ${auth} ${verb} ${path}`);
```

Comment:
The level that's currently being logged is `info`, so moving these up a level removes them from the default output. But if someone wants to turn the messages back on (which can be genuinely helpful for debugging), they can change the level.